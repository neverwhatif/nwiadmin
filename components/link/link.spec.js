import React from 'react';
import { shallow } from 'enzyme';

import Link from './index';

describe('Link', () => {
    const component = (<Link to="lorem">Lorem</Link>);

    it('should render', () => {
        expect(shallow(component)).toMatchSnapshot();
    });
});
