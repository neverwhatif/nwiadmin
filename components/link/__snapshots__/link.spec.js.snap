// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Link
    className={null}
    to="lorem"
  >
    Lorem
  </Link>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": "Lorem",
      "className": null,
      "replace": false,
      "to": "lorem",
    },
    "ref": null,
    "rendered": "Lorem",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Lorem",
        "className": null,
        "replace": false,
        "to": "lorem",
      },
      "ref": null,
      "rendered": "Lorem",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
